public/files/js/types/concordance.ts

```diff
@@ -25,15 +25,20 @@ export interface ConcToken {
 }
 
 
-export class TextChunk {
-    id:string;
+export class TextChunkBase implements ConcToken {
```

Comment:
This is controversial and perhaps an overkill. I could have just added `viewAttrs` to `ConcToken` and be done with it.